A bunch of small fixes

stv0g opened the pull request:
Hi @konstin,

this are a few commits fixing some broken translations, icons and inconsistencies in the code-style.

konstin commented on importer/import_json.py:
```diff
@@ -41,8 +41,17 @@
     ],
-    models.Consultation: ["meeting_id", "paper_id", "oparl_id"],
-    models.File: ["name", "oparl_download_url", "oparl_access_url", "oparl_id"],
+    models.Consultation: [
```
meine-stadt-transparent uses [black](https://github.com/psf/black), I've added a ci job in 97fe124b8cfc3fd7a54ad0f46ca3cdf4b239b128 to lint for that

konstin commented:
thanks for updating the translations and the icons

I think the renamed commands needs to also be changed in the docs, e.g. `citytoags` is still written that way in Import.md

stv0g commented:
Thanks for the tips,

I fixed the invalid references to the renamed commands and used black for checking the formatting.

The commits are also rebased on the latest main branch, so the new linting CI check is included.

konstin commented:
Thank you!
